src/web/src/components/SignUp/Schema/FormSchema.tsx

```diff
@@ -21,14 +21,14 @@ export default [
   Yup.object().shape({}),
 
   Yup.object().shape({
-    [firstName.name]: Yup.string().required(`${firstName.requiredErrorMsg}`),
-    [lastName.name]: Yup.string().required(`${lastName.requiredErrorMsg}`),
-    [displayName.name]: Yup.string().required(`${displayName.requiredErrorMsg}`),
+    [firstName.name]: Yup.string().required(firstName.requiredErrorMsg),
+    [lastName.name]: Yup.string().required(lastName.requiredErrorMsg),
+    [displayName.name]: Yup.string().required(displayName.requiredErrorMsg),
   }),
 
   // Second step we fetch data from GitHub.
   Yup.object().shape({
-    [githubUsername.name]: Yup.string().required(`${githubUsername.requiredErrorMsg}`),
+    [githubUsername.name]: Yup.string().required(githubUsername.requiredErrorMsg),
     [github.name]: Yup.object()
       .shape({
```

Comment:
So you're thinking to keep it as an Object, but just send it in the correct format to/from the server vs. splitting these apart here as well?

Reply:
Right, because there are 2 GitHub usernames, one is the user input, one is the one we get from GitHub, for displaying under the avatar. If we use just one, then the name under the avatar will change on input change and it doesn't match the avatar until we click `Get GitHub Profile`.